src/types/planning-constraints.ts

```diff
@@ -27,8 +34,8 @@ export type Constraint = {
   fn: string;
   value: boolean;
   text?: string;
-  data?: Array<Record<string, unknown>>; // `Array<MinimumDigitalLandEntity | MinimumOSRoadFeature>` in future??
-  category?: string;
+  data?: Array<Record<string, unknown>>; // @todo `Array<MinimumDigitalLandEntity | MinimumOSRoadFeature>`
+  category?: string; // `PlanningConstraintCategory`
```

Comment:
Setting this as the `PlanningConstraintCategory` string enum created a lot of tedious MUI List/Accordion type issues in planx-new because this prop serves as the header of each sublist. So it's a plain `string` for now, one to come back to sometime!